lib/main.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'onlyoffice_s3_wrapper'
+require_relative '../data/static_data'
+
+
+# Methods for download files
+class Downloader
+  def initialize
+    FileUtils.makedirs('./tmp')
```

Comment:
I think those lines can be switched and you can call `FileUtils.makedirs(@tmp_dir)` - so if we change this location in future - we need to change only one line